v2/openapi/fb-provider-crosstransfer-api.yaml

```diff
@@ -0,0 +1,750 @@
+      required: [ orderType, payment, order ]
+      properties:
+        id:
+          type: string
+          example: '76a33ed9-8aa2-4781-93f8-d2369c0c2b14'
+        orderType:
+          type: string
+          enum: [ OffRamp ]
+        payment:
+          allOf:
+            - $ref: '#/x-schemas/PublicBlockchainTransaction'
+            - type: object
+              required: [ status ]
+              properties:
+                status:
+                  $ref: '#/x-schemas/DepositStatus'
+        order:
+          allOf:
+            - $ref: '#/x-schemas/FiatTransfer'
+            - type: object
+              required: [ status ]
+              properties:
+                status:
+                  $ref: '#/x-schemas/WithdrawalStatus'
+
+    BridgingCrossTransfer:
+      type: object
+      required: [ orderType, payment, order ]
+      properties:
+        id:
+          type: string
+          example: '76a33ed9-8aa2-4781-93f8-d2369c0c2b14'
+        orderType:
+          type: string
+          enum: [ Bridging ]
+        payment:
+          allOf:
+            - $ref: '#/x-schemas/PublicBlockchainTransaction'
+            - type: object
+              required: [ status ]
+              properties:
+                status:
+                  $ref: '#/x-schemas/DepositStatus'
+        order:
+          allOf:
+            - $ref: '#/x-schemas/PublicBlockchainTransaction'
+            - type: object
+              required: [ status ]
+              properties:
+                status:
+                  $ref: '#/x-schemas/WithdrawalStatus'
+        
+    CryptoPaymentAccountParams:
+      allOf:
+        - $ref: '#/x-schemas/NativeCryptocurrency'
+        - type: object
+          required: [ developerFee ]
+          properties:
+            developerFee:
+              $ref: './fb-provider-api.yaml#/components/schemas/PositiveAmount'
+
+    FiatPaymentAccountParams:
+      allOf:
+        - $ref: '#/x-schemas/NationalCurrency'
+        - type: object
+          required: [ developerFee ]
+          properties:
+            developerFee:
+              $ref: './fb-provider-api.yaml#/components/schemas/PositiveAmount'
+
+    CreateBridgingRequest:
+      type: object
+      required: [ idempotencyKey, source, destination, ]
+      properties:
+        idempotencyKey:
+          type: string
+          example: '76a33ed9-8aa2-4781-93f8-d2369c0c2b14'
+        customerId:
+          type: string
+          description: If provided, the standing order will be associated with the customer.
+          example: '76a33ed9-8aa2-4781-93f8-d2369c0c2b14'
+        source:
+          $ref: '#/components/schemas/CryptoPaymentAccountParams'
+        destination:
+          $ref: '#/x-schemas/PublicBlockchainAddress'
+
+    ListCrosstransferStandingOrdersResult:
+      type: object
+      properties:
+        standingOrders:
+          type: array
+          items: 
+            $ref: '#/components/schemas/CrosstransferStandingOrder'
```

Comment:
What's the point of having a shell wrap for the list instead of just returning a list of `CrosstransferStandingOrder`